certified-connectors/GroupMgr/apiDefinition.swagger.json

```diff
@@ -1,982 +1,987 @@
 {
-   "swagger":"2.0",
-   "info":{
-      "title":"GroupMgr",
-      "description":"A powerful set of tools, for both IT managers and users, to efficiently create, manage, browse and analyse all groups in your Microsoft 365 environment.",
-      "version":"2.4",
-      "contact":{
-         "name":"GroupMgr Helpdesk",
-         "url":"https://www.vanroey.be/en/support/",
-         "email":"support@groupmgr.com"
-      }
-   },
-   "host":"vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
-   "basePath":"/",
-   "schemes":[
-      "https"
-   ],
-   "x-ms-connector-metadata":[
-      {
-         "propertyName":"Website",
-         "propertyValue":"https://groupmgr.com/"
-      },
-      {
-         "propertyName":"Privacy policy",
-         "propertyValue":"https://groupmgr.com/privacy"
-      },
-      {
-         "propertyName":"Categories",
-         "propertyValue":"IT Operations;Productivity"
-      }
-   ],
-   "consumes":[
-      
-   ],
-   "produces":[
-      
-   ],
-   "paths":{
-      "/api/webhookrequest/GroupRequested":{
-         "x-ms-notification-content":{
-            "description":"Caught group request.",
-            "schema":{
-               "$ref":"#/definitions/GroupCompact"
+    "swagger": "2.0",
+    "info": {
+        "title": "GroupMgr",
+        "description": "A powerful set of tools, for both IT managers and users, to efficiently create, manage, browse and analyse all groups in your Microsoft 365 environment.",
+        "version": "2.5",
+        "contact": {
+            "name": "GroupMgr Helpdesk",
+            "url": "https://www.vanroey.be/en/support/",
+            "email": "support@groupmgr.com"
+        }
+    },
+    "host": "vragroupmanagerapprovalflow-h8tvt-uzi02fbsspubp3eg.azurewebsites.net",
```

Comment:
@Jwaegebaert Can you please make sure that you are providing a valid host URL as we do not certify a connector with test or dev host URL.

Reply:
This URL is the one from production. It isn't a test or dev URL.